ci/build_container/build_recipes/benchmark.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+
+export COMMIT="e1c3a83b8197cf02e794f61228461c27d4e78cfb"  # benchmark @ Jan 11, 2018
```

Comment:
FYI, the new preferred way to add external deps is https://github.com/envoyproxy/envoy/blob/master/bazel/EXTERNAL_DEPS.md#adding-external-dependencies-to-envoy-genrule-repository.

Reply:
OK, I'll continue to iterate on that.